src/test/java/co/com/bancolombia/task/GenerateDrivenAdapterTaskTest.java

```diff
@@ -412,7 +412,7 @@ public void generateDrivenAdapterRedisRepositoryForImperative()
             .exists());
   }
 
-  @Test
+  @Test(expected = ValidationException.class)
   public void generateDrivenAdapterRedisRepositoryForReactiveWithSecret()
```

Comment:
should this test generate an exception?

Reply:
Same that in java

Reply:
but maybe the test name will be different

Reply:
but maybe the test name will be different